DEVELOPING.md

```diff
@@ -0,0 +1,60 @@
+# Developing Klaxon Locally #
+
+If you're interested in [helping improve Klaxon by contributing code](CONTRIBUTING.md), this is a quick guide to how to get it running on your local machine for development.
+
+First, we'll assume you already have git on your local machine, as well as a standard postgres installation. So go to your terminal, navigate to whatever directory where you keep your projects and clone the Klaxon repo.
```

Comment:
Might be worth mentioning and linking to https://postgresapp.com/ as an easy way to get Postgres up and running.